### ISSUE TITLE
Xcode 13.3 Enum Parsing Fails

### ISSUE THREAD

**Issue by StratusHunter** [realm/jazzy#1302]
Since installing Xcode 13.3 I can no longer run jazzy to make my documentation.

It will fail when attempting to parse a swift enum.

```
Parsing PermissionStateType.swift (19/52)
        from /Users/terencebaker/.rvm/gems/ruby-3.0.0/gems/jazzy-0.14.1/lib/jazzy/sourcekitten.rb:269:in 'run_sourcekitten'
        from /Users/terencebaker/.rvm/gems/ruby-3.0.0/gems/jazzy-0.14.1/lib/jazzy/doc_builder.rb:82:in 'block in build'
        from /Users/terencebaker/.rvm/gems/ruby-3.0.0/gems/jazzy-0.14.1/lib/jazzy/doc_builder.rb:80:in 'chdir'
        from /Users/terencebaker/.rvm/gems/ruby-3.0.0/gems/jazzy-0.14.1/lib/jazzy/doc_builder.rb:80:in 'build'
        from /Users/terencebaker/.rvm/gems/ruby-3.0.0/gems/jazzy-0.14.1/bin/jazzy:16:in '<top (required)>'
        from /Users/terencebaker/.rvm/gems/ruby-3.0.0/bin/jazzy:23:in 'load'
```

The project I'm trying to document is a local swift package written entirely in swift.

My config is:
```
output: docs
clean: true
min_acl: private
swift_build_tool: xcodebuild
build_tool_arguments:
  - -scheme
  - Quickstart
  - -sdk
  - iphonesimulator
  - -destination
  - "platform=iOS Simulator,name=iPhone 13"
 ```

`jazzy version: 0.14.1`

**Comment by johnfairh** [comment]
Fixed in master via #1298.

Should be a new version of the jazzy gem later today.

**Comment by StratusHunter** [comment]
Fantastic, thank you!
